perf/get.go

```diff
@@ -37,7 +37,7 @@ func (opts GetOptions) Validate() error {
 }
 
 func (opts GetOptions) parse() string {
-	urlString := fmt.Sprintf("%s/rest/v1/test_results/task_id/%s/%d/count", opts.Cedar.BaseURL, url.PathEscape(opts.TaskID), utility.FromIntPtr(opts.Execution))
+	urlString := fmt.Sprintf("%s/rest/v1/perf/task_id/%s/%d/count", opts.Cedar.BaseURL, url.PathEscape(opts.TaskID), utility.FromIntPtr(opts.Execution))
```

Comment:
We should enable both `perf/task_id/{task_id}` and `perf/task_id/{task_id}` routes. This can be done by adding the `Count` boolean field to the options.

Reply:
Also, the count route you wrote does not accept an execution: https://github.com/evergreen-ci/cedar/blob/main/rest/perf_routes.go#L187-L191.